crypto/bn/asm/x86_64-mont5.pl

```diff
@@ -2752,6 +2761,7 @@
 	mov	8*2(%rbp),%r14
 	mov	8*3(%rbp),%r15
 	jmp	.Lsqrx4x_sub_entry	# common post-condition
+.cfi_endproc
```

Comment:
This look wrong at first sight, I don't see a ret.

Reply:
consider it somehing like a tail-call, so
the function containing .Lsqrx4x_sub_entry does the ret

Reply:
I think this is already covered by the one at the end of __bn_postx4x_internal

Reply:
I guess I don't know enough about how this is supposed to work

Reply:
Right, as soon as the control flow enters __bn_postx4x_internal, the
cfi info is correct, just between mulx4x_internal and here, the
unwinder would need to guess.

Reply:
when gdb walks a function without any cfi instruction, it can try to
load the return address from sp[0] and catch any faults in accessing
those frames, the stack walking code in https://github.com/openssl/openssl/pull/10635#issue-353973686

I do basically just use this instrumented version of the speed command
and look at the stack traces from `opensssl speed` where they do not end
at PC=0